[FIX] Reword front page (#958)
bids-standard/bids-specification: src/index.md

# The Brain Imaging Data Structure

The Brain Imaging Data Structure (BIDS) is a simple and intuitive way to
organize and describe data.

> This document defines the BIDS specification, which provides all of the details
> to implement the standard. It includes the core specification as well as many extensions

Reviewer: ```suggestion
This document defines the BIDS specification, which provides many details
to help implement the standard. It includes the core specification as well as many extensions
```

I will split hair on this one, but unfortunately the spec does not provide "all" the details. I mean I wished it did otherwise we would not have to regularly open that many PR to add more details regarding some blind spots we had not thought of.

Similarly the spec still leave a lot of flexibility on how to implement it on a dataset.

So I am just "hedging" that sentence a bit.

Author: Fair enough :-)

to specific brain imaging modalities, and increasingly also to other kinds of
data.

If BIDS is new to you, and you would like to learn more about how to adapt your
own datasets to match the BIDS specification, we recommend exploring the
[BIDS Starter Kit](https://bids-standard.github.io/bids-starter-kit/).

> Alternatively, to get started please read [the introduction to the spec](01-introduction.md).

Reviewer: ```suggestion
Alternatively, to get started please read [the introduction to the specification](01-introduction.md).
```
